lib/engine/game/g_18_cz/game.rb

```diff
@@ -2756,7 +2756,7 @@ def rust_trains!(train, entity)
             next unless rust?(t)
 
             rusted_trains << t.name
-            owners[t.owner.name] += 1
+            owners[t.owner.name] += 1 unless t.owner.nil?
```

Comment:
if t.owner

Reply:
When I remove .nil? the tests fail for some reason =(

Reply:
Oh i am an idiot...